.github/actions/install-python-and-package/action.yml

```diff
@@ -59,26 +61,7 @@ runs:
       run: |
         # Install dependencies not handled by setuptools
         ## DSSP
-        git clone https://github.com/PDB-REDO/libcifpp.git --recurse-submodules
-        cd libcifpp
-        cmake -S . -B build -DCMAKE_INSTALL_PREFIX=$HOME/.local -DCMAKE_BUILD_TYPE=Release
-        cmake --build build
-        cmake --install build
-        #######
-        git clone https://github.com/mhekkel/libmcfp.git
-        cd libmcfp
-        mkdir build
-        cd build
-        cmake ..
-        cmake --build .
-        cmake --install .
-        #######
-        git clone https://github.com/PDB-REDO/dssp.git
-        cd dssp
-        mkdir build
-        cmake -S . -B build -DCMAKE_BUILD_TYPE=Release
-        cmake --build build
-        cmake --install build
+        conda install -c sbl dssp>=4.2.2.1
```

Comment:
This is for `if runner.os == 'macOS'`, while the one in line 48 is for linux. We're not running the CI on mac yet, but I keep it updated since it's in my todo list